### PR TITLE
Remove level tags from atomic judgments.

### PR DESCRIPTION
**Pull request by favonia**
@jonsterling You might want to check whether this is still on the common path shared by us.

### CODE REVIEW

**Comment by jonsterling** [comment]
this sounds just like what I have in mind. good!

File: src/redprl/refiner_kit.fun [hunk 1]
```diff
@@ -207,6 +207,17 @@ struct
     if Abt.eq (a, b) then NONE
     else SOME @@ makeSubType H (a, b)
 
+  (* functions which blur the difference between EQ and EQ_TYPE *)
+  val viewAsEqType =
```
Comment (favonia) [inline src/redprl/refiner_kit.fun]: Manual "view as" pattern matching.

File: src/redprl/refiner_types.fun [hunk 2]
```diff
@@ -56,10 +56,10 @@ struct
     fun EqType _ jdg =
       let
         val _ = RedPrlLog.trace "Bool.EqType"
-        val H >> AJ.EQ_TYPE ((a, b), l, k) = jdg
+        val H >> ajdg = jdg
+        val ((a, b), _, k) = viewAsEqType ajdg
```
Comment (favonia) [inline src/redprl/refiner_types.fun]: @jonsterling So this is the new programming style to handle both `EQ` and `EQ_TYPE` at once.

Comment (jonsterling) [reply src/redprl/refiner_types.fun]: cool! That sounds reasonable.

**Comment by favonia** [comment]
Hmm let me merge the coequalizers...

**Comment by favonia** [comment]
@jonsterling I introduced non-deterministic auto-synth in `auto`. This way we can `claim` some well-typedness.

**Comment by jonsterling** [comment]
> @jonsterling I introduced non-deterministic auto-synth in auto. This way we can claim some well-typedness.

Interesting! Is there an example of where this fires? (If not, you don't need to come up with one, I was just curious.)

**Comment by favonia** [comment]
> Is there an example of where this fires?

Somehow I suddenly need this in `semi-simplicial.prl`. See the addition of `aux2` in two places. If we can avoid it that would be great!

**Pull request merged**